lib/diagnostic.js

```diff
@@ -5,22 +5,36 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 const min = function min() {
-
+  let result = arguments[0];
+    for( let i = 1; i < arguments.length; i++){
+      if(arguments[i] < result){
+        result = arguments[i];
+      }
+    }
+    return result;
 };
 
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
 const sum = function sum() {
-
+  let result = 0;
+    for( let i = 0; i < arguments.length; i++){
+        result = result + arguments[i];
+    }
+    return result;
```

Comment:
Does not pass the test when no argument is passed into the `sum` function. You may need an `if` statement to check if a argument is passed.